config/core/configmaps/leader-election.yaml

```diff
@@ -21,7 +21,7 @@ metadata:
     eventing.knative.dev/release: devel
     knative.dev/example-checksum: "43edfe90"
   annotations:
-    knative.dev/example-checksum: f7320456
+    knative.dev/example-checksum: 6cf53e10
```

Comment:
Can you quote this? And also while here, would you mind removing the label as it appears to have been duplicated in the past and is wrong :)

Reply:
I can remove the old, but I'd rather not quote it manually as this is populated by code-gen.  cc @markusthoemmes

Reply:
Pretty please? :) 
https://github.com/knative/pkg/pull/1472

Or did I misunderstand that PR?

Reply:
Looks like Weston already did this.